### ISSUE TITLE
Focus style and summary ::marker indicator on Firefox

### ISSUE THREAD

**Issue by dsamojlenko** [cds-snc/c19-benefits-node#344]
From ESDC a11y review, May 1:

> WCAG 2.0: When the “Report a problem or mistake on this page” summary receives keyboard focus, no focus outline appears around it. I’d recommend adding a focus outline to it.

> WCAG 2.0: “Report a problem or mistake on this page” is coded as a summary for a <details> element, but doesn’t look like one. I’d recommend adding a triangle just before the text it to make it obvious that it’s expandable/collapsible content (i.e. a triangle that points rightwards when collapsed and changes to point downwards when expanded).
